packages/react/src/components/date-picker/date-picker.decorator.tsx

```diff
@@ -0,0 +1,16 @@
+import { LabelParameters } from '@dso-toolkit/sources';
+import * as React from 'react';
+
+export const decorator: LabelParameters<JSX.Element>['decorator'] = (story) => (
+  <>
+    <div className='narrow-input'>
+      {story()}
+    </div>
+
+    <style>
+      .narrow-input {'{'}
+        width: 175px;
+      {'}'}
+    </style>
+  </>
```

Comment:
Dit is een beetje rommelig en komt ook niet overeen met de decorator zoals je 'm hebt gemaakt in `lit-html`.

Probeer eens iets als dit:

```suggestion
  <div style={({ width: '175px' })}>
    {story()}
  </div>
```

Reply:
AH ja, ik zat te stoeien met de syntax.

Reply:
Maar heb je het nou bijgewerkt? 🤔

Reply:
Ik heb 'm lokaal toegepast en ga zo mergen